src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -58,13 +59,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       return;
     }
 
-    // TODO(JW): Merge "the creation process" from here to UserStore
     String password = request.getParameter("password");
-    String hashedPassword = BCrypt.hashpw(password, BCrypt.gensalt());
-
-    User user = new User(UUID.randomUUID(), username, hashedPassword, Instant.now());
-    userStore.addUser(user);
 
+    userStore.addUser(username, password, false);
+    //String hashedPassword = BCrypt.hashpw(password, BCrypt.gensalt());
```

Comment:
You can get rid of these commented out lines.

Reply:
Ok, fixed!